src/web/app/pages-monitoring/logs-page/logs-page.component.html

```diff
@@ -36,12 +36,12 @@ <h1>Logs</h1>
               </button>
             </div>
           </div>
-          <div id="logs-to-timepicker" class="col-md-4">
+          <div id="logs-to-timepicker" class="col-md-4 mb-2">
```

Comment:
Just a nit. Let's follow the `from-timepicker`'s margin too, i.e., It is using `mt-4`, so that it looks consistent 😄 
```suggestion
          <div id="logs-to-timepicker" class="col-md-4 mb-4">
```

Reply:
Sorry I don't understand, should I add a `mt-4` instead of `mb-4`?

Reply:
Sorry for the confusion. Please add `mb-4`!